concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
         )
 
 
-def calculate_timestamp(current_time, concent_messaging_time):
+def calculate_timestamp(current_time, constants):
     return timestamp_to_isoformat(
-        current_time - (2 + _precalculate_subtask_verification_time(concent_messaging_time))
+        current_time - (2 * constants.concent_messaging_time + _precalculate_subtask_verification_time(constants))
     )
 
 
-def calculate_deadline(current_time, concent_messaging_time):
-    return current_time - (1 + _precalculate_subtask_verification_time(concent_messaging_time))
+def calculate_deadline(current_time, constants):
+    return current_time - (constants.concent_messaging_time + _precalculate_subtask_verification_time(constants))
 
 
-def calculate_deadline_too_far_in_the_future(current_time, concent_messaging_time):
-    return current_time - (1 + (20 * _precalculate_subtask_verification_time(concent_messaging_time)))
+def calculate_deadline_too_far_in_the_future(current_time, constants):
+    return current_time - (1 + (20 * _precalculate_subtask_verification_time(constants)))
 
 
-def _precalculate_subtask_verification_time(concent_messaging_time):
+def _precalculate_subtask_verification_time(constants):
```

Comment:
I would insist on changing this function signature to accept only required parameters.

Reply:
Done